### DISCUSSION TITLE
Facing error in the command for training a single GPU in 3D Latent Diffusion Model on Brats data example

### DISCUSSION THREAD

**Discussion by CSpoorthi** [Project-MONAI/MONAI#6678] [Q&A] [answered]
We followed the steps from the readme of 3D Latent Diffusion Model on Brats data 
We were able to download the dataset using the command "python download_brats_data.py -e ./config/environment.json"
also we followed the steps from Installation of MONAI generative model
In step 3 where we need to train the autoencoder for a single GPU using the command "python train_autoencoder.py -c ./config/config_train_32g.json -e ./config/environment.json -g 1" 
Since our machine GPU cuda capability is 3.5, we intsalled PyTorch of version 1.9.1 (Linux ubuntu 20.04)
Here we are getting the error : `python3 train_autoencoder.py -c ./config/config_train_32g.json -e ./config/environment.json -g 1`

```
Traceback (most recent call last):
  File "train_autoencoder.py", line 21, in <module>
    from generative.losses import PatchAdversarialLoss, PerceptualLoss
  File "/home/spoorthi/.local/lib/python3.8/site-packages/generative/losses/_init_.py", line 15, in <module>
    from .perceptual import PerceptualLoss
  File "/home/spoorthi/.local/lib/python3.8/site-packages/generative/losses/perceptual.py", line 17, in <module>
    from torchvision.models import ResNet50_Weights, resnet50
ImportError: cannot import name 'ResNet50_Weights' from 'torchvision.models' (/home/spoorthi/.local/lib/python3.8/site-packages/torchvision/models/_init_.py)
```
Can anyone give any solution for this error which is compatible with our machine's configuration

**Comment by wyli** [comment]
since the pytorch version is 1.9.1, perhaps you should confirm the torchvision version is 0.10.1 `pip install torchvision==0.10.1` and maybe clear the model weight cache folder located at  `python -c "import torch; print(torch.hub.get_dir())"`

**Reply by CSpoorthi** [reply]
The torchvision version is 0.10.1 and I cleared the cache but still the error exists

**Reply by wyli** [reply, marked as answer]
I see, it seems `ResNet50_Weights` is introduced in torchvision 0.13 https://github.com/pytorch/vision/releases/tag/v0.13.0, the model can only run in torch 1.12 + torchvision 0.13 or newer.

**Reply by CSpoorthi** [reply]
Oh okay, thanks for providing the information.
